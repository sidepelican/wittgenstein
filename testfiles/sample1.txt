you can replace between WITTGENSTEIN_BEGIN and WITTGENSTEIN_END with shell command result.

# WITTGENSTEIN_BEGIN `echo hello`
must be overriden here
# WITTGENSTEIN_END
world
