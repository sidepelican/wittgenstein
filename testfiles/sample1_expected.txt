you can replace between WITTGENSTEIN_BEGIN and WITTGENSTEIN_END with shell command result.

# WITTGENSTEIN_BEGIN `echo hello`
hello
# WITTGENSTEIN_END
world
